### PR TITLE
[Merged by Bors] - chore(SetTheory/Cardinal/Basic): `Cardinal.IsLimit` → `IsSuccLimit`

### PR DESCRIPTION
**Pull request by vihdzp**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Cardinal/Basic.lean [hunk 1]
```diff
@@ -716,27 +716,43 @@ theorem add_one_le_succ (c : Cardinal.{u}) : c + 1 ≤ succ c := by
   cardinal by this definition, but `0` isn't.
 
 TODO: deprecate this in favor of `Order.IsSuccLimit`. -/
```
Comment (urkud) [inline Mathlib/SetTheory/Cardinal/Basic.lean:718]:
```suggestion
Deprecated. Use `Order.IsSuccLimit` instead. -/
```

File: Mathlib/SetTheory/Cardinal/Basic.lean [hunk 2]
```diff
@@ -886,9 +901,19 @@ lemma exists_eq_of_iSup_eq_of_not_isSuccPrelimit
   rw [iSup, csSup_of_not_bddAbove hf, csSup_empty]
   exact isSuccPrelimit_bot
 
-@[deprecated exists_eq_of_iSup_eq_of_not_isSuccPrelimit (since := "2024-09-05")]
-alias exists_eq_of_iSup_eq_of_not_isSuccLimit := exists_eq_of_iSup_eq_of_not_isSuccPrelimit
+lemma exists_eq_of_iSup_eq_of_not_isSuccLimit
+    {ι : Type u} [hι : Nonempty ι] (f : ι → Cardinal.{v}) (hf : BddAbove (range f))
+    {c : Cardinal.{v}} (hc : ¬ IsSuccLimit c)
+    (h : ⨆ i : ι, f i = c) : ∃ i, f i = c := by
```
Comment (YaelDillies) [inline Mathlib/SetTheory/Cardinal/Basic.lean:907]:
```suggestion
    (h : ⨆ i, f i = c) : ∃ i, f i = c := by
```

**Comment by YaelDillies** [comment]
Thanks!

maintainer merge

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**